include/msgpack/v1/adaptor/cpp11/chrono.hpp

```diff
@@ -30,6 +27,97 @@ MSGPACK_API_VERSION_NAMESPACE(v1) {
 
 namespace adaptor {
 
+namespace detail {
+template<typename Target,
+         typename Source,
+         bool target_is_signed = std::is_signed<Target>::value,
+         bool source_is_signed = std::is_signed<Source>::value,
+         typename = typename std::enable_if<
+                      std::is_integral<Target>::value &&
+                      std::is_integral<Source>::value
+                    >::type>
```

Comment:
```suggestion
template <
    typename Target,
    typename Source,
    bool target_is_signed = std::is_signed<Target>::value,
    bool source_is_signed = std::is_signed<Source>::value,
    typename = typename std::enable_if<
        std::is_integral<Target>::value &&
        std::is_integral<Source>::value
    >::type
>
```

Just coding style request:

1. Please add one whitespace between `template` and `<`.
2. Please use the same rule as (non template parameters) for template parameters.
3. Please set the same level `xxxx <` and `>`.

In addition, please use the same rule that I requested at the #1075.

Reply:
I don't understand the 2nd point, could you give an example?

Reply:
Ok, 2nd point.

# For non template function parameters:

## OK

```cpp
ret_type func_name(
    int param1,
    char* param2,
    some_type const& param3
) {
    // ...
}
```

```cpp
ret_type func_name(
    int param1,
    char* param2,
    some_type const& param3) {
    // ...
}
```

## NG

```cpp
ret_type func_name(int param1,
                   char* param2,
                   some_type const& param3
) {
    // ...
}
```

Ok, 2nd point.

# Fortemplate function parameters:


## OK

```cpp
template <typename T, typename U>
ret_type func_name(...) {
}
```

```cpp
template <
    typename T,
    typename U
>
ret_type func_name(...) {
}
```
## NG

```cpp
template <typename T,
          typename U
>
ret_type func_name(...) {
}
```

Reply:
got it, thank you